gpu-operator/gpu-operator-rdma.rst

```diff
@@ -22,52 +22,80 @@ GPUDirect RDMA and GPUDirect Storage
-`CUDA 11.4 and R470 drivers <https://docs.nvidia.com/cuda/gpudirect-rdma/index.html#changes-in-cuda-11-4>`_, a
-new kernel module ``nvidia-peermem`` is included in the standard NVIDIA driver installers (e.g. ``.run``). The
-kernel module provides Mellanox Infiniband-based HCAs direct peer-to-peer read and write access to the GPU's memory.
+To support GPUDirect RDMA, userspace CUDA APIs are required.
+The kernel mode support is provided by one of two approaches: DMA-BUF from the Linux kernel or the legacy ``nvidia-peermem`` kernel module.
+NVIDIA recommends using the DMA-BUF rather than using the ``nvidia-peermem`` kernel module from the GPU Driver.
 
 Starting with v23.9.1 of the Operator, the Operator uses GDS driver version 2.17.5 or newer.
 This version and higher is only supported with the NVIDIA Open GPU Kernel module driver.
 The sample commands for installing the Operator include the ``--set useOpenKernelModules=true``
 command-line argument for Helm.
 
-In conjunction with the `Network Operator <https://github.com/Mellanox/network-operator>`_, the GPU Operator can be used to
-set up the networking related components such as Mellanox drivers, ``nvidia-peermem`` and Kubernetes device plugins to enable
-workloads to take advantage of GPUDirect RDMA and GPUDirect Storage. Refer to the Network Operator `documentation <https://docs.nvidia.com/networking/display/COKAN10>`_
-on installing the Network Operator.
+In conjunction with the Network Operator, the GPU Operator can be used to
+set up the networking related components such as network device kernel drivers and Kubernetes device plugins to enable
+workloads to take advantage of GPUDirect RDMA and GPUDirect Storage.
+Refer to the Network Operator `documentation <https://docs.nvidia.com/networking/software/cloud-orchestration/index.html>`_ for installation information.
 
-***********************************************
-Configuring GPUDirect RDMA Using nvidia-peermem
-***********************************************
+********************
+Common Prerequisites
+********************
 
-Platform Support
-================
+The prerequisites for configuring GPUDirect RDMA or GPUDirect Storage depend on whether you use DMA-BUF from the Linux kernel or the legacy ``nvidia-peermem`` kernel module.
 
-The following platforms are supported for GPUDirect with RDMA:
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 20 40 40
 
-* Kubernetes on bare metal and on vSphere VMs with GPU passthrough and vGPU.
-* VMware vSphere with Tanzu.
-* For Red Hat Openshift on bare metal and on vSphere VMs with GPU passthrough and vGPU configurations,
-  see the :ref:`NVIDIA AI Enterprise with OpenShift <nvaie-ocp>` information.
+   * - Technology
+     - DMA-BUF
+     - Legacy Nvidia-peermem
+
+   * - GPU Driver
+     - Branch 515, 535 or higher.
+       An Open Kernel module driver is required.
+     - Branch 470, 535 or higher.
```

Comment:
I wonder if we should mention R470 as it is EOL as of Aug 2024

Reply:
I see your point.  The only downside to removing it, is that it diminishes the impact of "legacy."

After checking our [support page](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/platform-support.html#gpu-operator-component-matrix), I'll remove 515 because it's not there.  When we remove 470, I think this row becomes "Open Kernel module driver is required." vs "Any supported GPU driver."

Reply:
I looked over this again and agree with you.  I removed mention of 470.  TY again.